.gitignore

```diff
@@ -17,7 +17,6 @@ target/*
 .idea/
 .lein-repl-history
 example.rb
-Dockerfile
```

Comment:
This shouldn't be added. You will add a docker file to the `docker` folder and add it to the `makefile`. Then you can run `make objc` to build the file